tsl/test/expected/cagg_drop_chunks.out

```diff
@@ -145,184 +145,152 @@ NOTICE:  drop cascades to 32 other objects
+    )
+    SELECT MIN(range_start) AS range_start, MAX(range_end) AS range_end
+    INTO _r
+    FROM
+        _chunks
+        JOIN _timescaledb_catalog.chunk ON chunk.schema_name = _chunks.nspname AND chunk.table_name = _chunks.relname
+        JOIN _timescaledb_catalog.chunk_constraint ON chunk_id = chunk.id
+        JOIN _timescaledb_catalog.dimension_slice ON dimension_slice.id = dimension_slice_id;
+
+    RAISE INFO 'range_start=% range_end=%', _r.range_start::int, _r.range_end::int;
+    CALL refresh_continuous_aggregate(_cagg, _r.range_start::int, _r.range_end::int + 1);
+END;
+$$
+LANGUAGE plpgsql;
+CREATE OR REPLACE FUNCTION test_int_now() returns INT LANGUAGE SQL STABLE as
     $$ SELECT 125 $$;
-CREATE TABLE conditions(time_int INT NOT NULL, device INT, value FLOAT);
-SELECT create_hypertable('conditions', 'time_int', chunk_time_interval => 10);
+CREATE TABLE conditions(time_int INT NOT NULL, value FLOAT);
+SELECT create_hypertable('conditions', 'time_int', chunk_time_interval => 4);
     create_hypertable    
 -------------------------
  (3,public,conditions,t)
 (1 row)
 
 INSERT INTO conditions
-SELECT time_val, time_val % 4, 3.14 FROM generate_series(0,100,1) AS time_val;
+SELECT time_val, 1 FROM generate_series(0, 19, 1) AS time_val;
 SELECT set_integer_now_func('conditions', 'test_int_now');
  set_integer_now_func 
 ----------------------
  
 (1 row)
 
-CREATE MATERIALIZED VIEW conditions_7
+CREATE MATERIALIZED VIEW conditions_2
     WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE)
     AS
-        SELECT time_bucket(7, time_int) as bucket,
+        SELECT time_bucket(2, time_int) as bucket,
             SUM(value), COUNT(value)
         FROM conditions GROUP BY bucket WITH DATA;
-NOTICE:  refreshing continuous aggregate "conditions_7"
+NOTICE:  refreshing continuous aggregate "conditions_2"
 HINT:  Use WITH NO DATA if you do not want to refresh the continuous aggregate on creation.
-CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket <= 70 ORDER BY bucket;
-SELECT * FROM see_cagg;
- bucket |  sum  | count 
---------+-------+-------
-      0 | 21.98 |     7
-      7 | 21.98 |     7
-     14 | 21.98 |     7
-     21 | 21.98 |     7
-     28 | 21.98 |     7
-     35 | 21.98 |     7
-     42 | 21.98 |     7
-     49 | 21.98 |     7
-     56 | 21.98 |     7
-     63 | 21.98 |     7
-     70 | 21.98 |     7
-(11 rows)
-
--- This is the simplest case, when the updated bucket is inside a chunk, so it is expected
--- that the update is refreshed.
-UPDATE conditions SET value = 4.00 WHERE time_int = 2;
--- This case updates values in the bucket, which affects two different partials in
--- two different chunks.
-UPDATE conditions SET value = 4.00 WHERE time_int = 9;
-UPDATE conditions SET value = 4.00 WHERE time_int = 11;
-SELECT timescaledb_experimental.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 20));
```

Comment:
Why do we remove those comments? Don't we test the same cases?

Reply:
It's not exactly the same because with the new implementation we can't test this in this way because the raw hypertable chunks are not aligned anymore with data stored into the materialization hypertable (we removed the chunk_id)